### PR TITLE
 Homework for JS1 week3

### PR DESCRIPTION
**Pull request by SvitlanaBurlaka**
Added homework for JS1 week3

### CODE REVIEW

File: javascript/javascript1/week3/homework3.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+for (let i = 0; i < names.length; i++) {
+    if (names[i] === nameToRemove) {
+        names.splice(i, 1);
+    }
+}
+// delete names[1];
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
```
Comment (vulk29) [inline javascript/javascript1/week3/homework3.js:22]: great job!

File: javascript/javascript1/week3/homework3.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    Math.floor(travelInformation.calcTime()) +
+    ` hours and ` +
+    Math.floor(
+        (travelInformation.calcTime() -
+            Math.floor(travelInformation.calcTime())) *
+        60
+    ) +
+    ` minutes`
+); // 8 hours and 38 minutes
+
```
Comment (vulk29) [inline javascript/javascript1/week3/homework3.js:43]: I like that you are playing with these things! great job!

File: javascript/javascript1/week3/homework3.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        seriesDurations[i].title +
+        " took " +
+        logOutSeriesText(seriesDurations[i]) +
+        "% of my life ."
+    );
+    totalTime = totalTime + logOutSeriesText(seriesDurations[i]);
+}
+
+console.log("In total that is " + totalTime + "%  of my life");
+
```
Comment (vulk29) [inline javascript/javascript1/week3/homework3.js:96]: Good! nice and clean!

File: javascript/javascript1/week3/homework3.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+saveNote("Do laundry", 2);
+saveNote("Make dinner", 3);
+
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+// Get a note
+
+function getNote(id) {
+    for (i = 0; i < notes.length; i++) {
+        if (notes.i === id) {
```
Comment (vulk29) [inline javascript/javascript1/week3/homework3.js]: if(nodes[i].id===id)

File: javascript/javascript1/week3/homework3.js [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+            return notes[i];
+        } else {
```
Comment (vulk29) [inline javascript/javascript1/week3/homework3.js]: if you want to use an if else combo here then what you can do is define a variable outside the for loop (e.g. ID_we_are_looking for), initialize it with e.g. -1, go through the for loop and check if any of the notes.ids matches the id you are looking for, if so, update the ID_we_are_looking_for with the id you found. 
IF at the end of the for loop, the ID_we_are_looking_for is still -1 THEN you can console.log the error message.

Makes sense? Give it  a go, and i will show you my solution if you get stuck! :)

File: javascript/javascript1/week3/homework3.js [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+            console.log("You dont have this note");
+        }
+    }
+}
+
+const firstNote = getNote();
```
Comment (vulk29) [inline javascript/javascript1/week3/homework3.js]: here you need to pass on a parameter> so it needs to be getNote(1) or getNote(2). your id on line 113 takes this value

File: javascript/javascript1/week3/homework3.js [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+            "The note with id : " +
+            notes[i].id +
+            " , has the following note text : " +
+            notes[i].content
+        );
+    }
+}
+
+logOutNotesFormatted();
+
```
Comment (vulk29) [inline javascript/javascript1/week3/homework3.js:143]: good job!

**Review by vulk29 — CHANGES_REQUESTED** [review]

File: javascript/javascript1/week3/homework3.js [hunk 8]
```diff
@@ -109,18 +109,21 @@ saveNote("Make dinner", 3);
...
     for (i = 0; i < notes.length; i++) {
-        if (notes.i === id) {
+        if (notes[i].id === id) {
+            neddedId = notes[i].id;
             return notes[i];
-        } else {
-            console.log("You dont have this note");
         }
     }
+    if (neddedId === -1) {
```
Comment (SvitlanaBurlaka) [inline javascript/javascript1/week3/homework3.js:121]: @vulk29 Thank you for review =)  can you look on my changes please? I hope i get what you meant

Comment (vulk29) [reply javascript/javascript1/week3/homework3.js:121]: yup! this is exactly what i had in mind! Excellent job!

**Review by vulk29 — APPROVED** [review]

**Pull request merged**